### ISSUE TITLE
Cozmo App doesn't connect to pycozmo server

### ISSUE THREAD

**Issue by reisxd** [zayfod/pycozmo#54]
Title says it all. And yes i modified Cozmos app to connect to my PC instead of cozmo (well... i don't even have a cozmo lol).

The engine sends "COZ\x03RE\x01" but the pycozmo server doesn't respond to it for some reason...?

Edit: If someone wants it, I can give pcap file from my phone.

**Comment by reisxd** [comment]
This is probably a library problem. I tried connecting to the server on my PC from my phone and it didn't work and tried the opposite. Even installed pycozmo on my RPI Zero and connecting to it from my phone/pc and it still didn't work.

@zayfod Do you have any idea why?

**Comment by reisxd** [comment]
Update again, modified Cozmos app once again to connect to the local server on my phone and it works (sort of, still doesn't go past "Connecting to Cozmo..." screen(I get the Connected message on the server though, might need to add missing packets)).

Why does this work locally?

**Comment by reisxd** [comment]
Probably the last reply if this doesn't get any response from anyone else.
I today came back to this and I found out the Cozmo app repeatedly sending the Ping frame and I think PyCozmo sends something invalid, causing Cozmo app to send Ping frame again and again until it fails.

Also, the server I think doesn't support giving cube data because I got to the "Pull the tabs on your Cubes" screen (by forcing the app) and clicking continue after it appears in a few seconds (Note that the frame might not even got sent because the Cozmo app probably closes the TCP connection after it fails).

**Comment by zayfod** [comment]
I am not sure I understand your setup.

PyCozmo is designed to interact with Cozmo and not the Cozmo mobile app.

**Comment by reisxd** [comment]
> I am not sure I understand your setup.
> 
> PyCozmo is designed to interact with Cozmo and not the Cozmo mobile app.

Well, I've figured it out a few weeks ago.
It was because, guess it, I was not sending packets to the client.
Sooo, it just failed at doing anything when it didn't receive the robot events (like that event with FW version, S/N etc).
I tried emulating as a robot but I still failed (don't know why, also discovered the app has a native library for communicating with the robot). So, I just gave up.
